backstop/tests/iModel-loader.html

```diff
@@ -0,0 +1,398 @@
+<!--
+  Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+  See LICENSE.md in the project root for license terms and full copyright notice.
+-->
+<!DOCTYPE html>
+<html
+  lang="en-US"
+  id="theme"
+>
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1"
+    >
+    <title>iModel loader | iTwinUI</title>
+    <link
+      rel="stylesheet"
+      href="../../lib/css/all.css"
+    />
+    <link
+      rel="stylesheet"
+      href="../assets/demo.css"
+    />
+    <style>
+      .iui-demo-full-page {
+        height: 90vh;
+        width: 100%;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+      }
+
+      .iui-loading-imodel {
+        width: 250px;
+        margin-bottom: 20px;
+      }
+    </style>
+    <script src="../assets/theme.js"></script>
+  </head>
+  <body class="iui-body">
+    <theme-button></theme-button>
+    <section class="iui-demo-full-page">
+      <div
+        id="imodel-loader"
+        class="iui-progress-indicator-full-page"
+      >
+        <!-- Use iModel-loader.svg from @itwin/itwinui-illustrations package. -->
+        <svg
+          class="iui-loading-imodel"
+          xmlns="http://www.w3.org/2000/svg"
+          viewBox="0 0 739.97 854.77"
+        >
+          <style>
+            .iui-loading-imodel-shape {
```

Comment:
style shouldnt be just hardcoded in the svg?
Idea is, that the svg below is the exact code you get from illustrations package.

Reply:
It is hardcoded in the exact code
https://github.com/iTwin/iTwinUI-icons/pull/21/files

Reply:
I added same svg code (with classes and style) in illustrations package. Should I just repeat `fill: color` for each polygon and remove style tag?

Reply:
I see there are also those classes in illustrations, so tahts ok, I guess.. just sure you want to keep unused classes there too?